tensorflow_datasets/audio/freesound_audio_tagging_2019.py

```diff
@@ -0,0 +1,152 @@
+
+# Train set
+
+The train set is meant to be for system development. The idea is to limit the supervision provided (i.e., the manually-labeled data), thus promoting approaches to deal with label noise. The train set is composed of two subsets as follows:
+
+##Curated subset
+
+The curated subset is a small set of manually-labeled data from FSD.
+
+    Number of clips/class: 75 except in a few cases (where there are less)
+    Total number of clips: 4970
+    Avge number of labels/clip: 1.2
+    Total duration: 10.5 hours
+
+The duration of the audio clips ranges from 0.3 to 30s due to the diversity of the sound categories and the preferences of Freesound users when recording/uploading sounds. It can happen that a few of these audio clips present additional acoustic material beyond the provided ground truth label(s).
+
+## Noisy subset
+
+The noisy subset is a larger set of noisy web audio data from Flickr videos taken from the YFCC dataset.
+
+    Number of clips/class: 300
+    Total number of clips: 19815
+    Avge number of labels/clip: 1.2
+    Total duration: ~80 hours
+
+The duration of the audio clips ranges from 1s to 15s, with the vast majority lasting 15s.
+
+Considering the numbers above, per-class data distribution available for training is, for most of the classes, 300 clips from the noisy subset and 75 clips from the curated subset, which means 80% noisy - 20% curated at the clip level (not at the audio duration level, considering the variable-length clips).
+'''
+
+_URLS = [
+    'http://dcase.community/challenge2019/task-audio-tagging',
+    'https://www.kaggle.com/c/freesound-audio-tagging-2019/',
+]
+
+
+class FreesoundAudioTagging2019Config(tfds.core.BuilderConfig):
+    def __init__(self, concatenate_sources=False, **kwargs):
+        self.concatenate_sources = concatenate_sources
+        super(FreesoundAudioTagging2019Config, self).__init__(**kwargs)
+
+
+class FreesoundAudioTagging2019(tfds.core.GeneratorBasedBuilder):
+
+    # TODO Is there an official semver version for the dataset?
+    VERSION = tfds.core.Version('1.0.0')
+
+    def _info(self):
+        audio = tfds.features.Text()
+        tags = tfds.features.FeaturesDict({name: tf.bool for name in _TAGS})
+        dataset_info = tfds.core.DatasetInfo(
+            builder=self,
+            description=_DESCRIPTION,
+            features=tfds.features.FeaturesDict({
+                'audio': audio,
+                'tags': tags
+            }),
+            supervised_keys=('audio', 'tags'),
+            urls=_URLS,
+            citation=_CITATION,
+        )
+        return dataset_info
+
+    def _split_generators(self, dl_manager):
+        download_dir = Path(dl_manager._download_dir)
+
+        csvs = {
+            tfds.Split.TRAIN: download_dir / 'train_noisy.csv',
```

Comment:
I think you miss quotes.

Use `os` library to be compatible with all environments.
Like that `os.path.join('download_dir', 'train_noisy.csv')`